tests/package_test.md

```diff
@@ -0,0 +1,50 @@
+# How to run the tests
+
+## Rrequirements
+
+Additionaly to the package building requirements the python nosetest running framework is required.
+
+~~~~~~{.sh}
+# source the already created python virtualenv for package building
+source ~/checker_env/bin/activate
+
+# install nosetest
+pip install nose
+~~~~~~
+
+## Running the tests
+
+To run the tests call the run_test.sh script.
+It will build a package, and run the all the unit and functional tests.
+~~~~~~{.sh}
+run_tests.sh
+~~~~~~
+
+## More detailed test running
+~~~~~~{.sh}
+# setup some required environment variables to run the tests
+# create temporary directories
+source tests/test_env_setup.sh
+
+# build the CodeChecker package
+build_package.py -o $TEST_CODECHECKER_PACKAGE_DIR -v
+
+# setup the environment to use SQLite database for testing
```

Comment:
In this step we require the user to set `TEST_USE_POSTGRESQL` and `CODECHECKER_DB_DRIVER` environment variables. The former one is quite descriptive, but the meaning of the latter one is not clear. When we provide commands to the user i.e. commands are not run by scripts, then those should be described.